### DISCUSSION TITLE
New Release/Version

### DISCUSSION THREAD

**Discussion by Kingproone** [audacious-media-player/audacious#133] [General]
So it has almost been a year since the last release and I was just wondering when a new release is planned?

**Comment by radioactiveman** [comment]
No real plan yet, but in my opinion we have enough improvements and fixes for a new release. I guess we can create as first step a beta release in April. Do you have some specific feature of Audacious 4.4 you are waiting for? :)

@jlindgren90: What's your opinion here?

**Reply by jlindgren90** [reply]
A new beta sounds good to me

**Reply by Kingproone** [reply]
Not a single feature, just the reassurement that the project is still ongoing and alive is great to hear!

**Reply by jlindgren90** [reply]
It is :) In my opinion it is quite stable though and thus there isn't a need for frequent releases so much as there used to be.

**Reply by kaniini** [reply]
Yeah at this point there really isn't much to do other than fix the occasional bug... trying to do feature-focused development at this point would just ruin what we've built.

**Reply by Kingproone** [reply]
Well, that's understandable, once a project has matured like this one, but a "housekeeping" update once a while still is a nice touch. Thanks to everyone who worked on this amazing audioplayer!

**Comment by radioactiveman** [comment]
Today I have released the 4.4-beta1. The stable release will follow soon, probably in a few weeks.

Changelog: https://github.com/audacious-media-player/audacious/releases/tag/audacious-4.4-beta1

@jlindgren90: Could you please take care of the Windows build again and reference it at https://audacious-media-player.org/download? If I remember it correctly, we also need to update the info about supported Windows versions here. Take your time though, it's not urgent. Thanks.

**Reply by jlindgren90** [reply]
Thanks so much for handling the release! I uploaded Windows builds just now but have run out of time to update the website. 64-bit Windows 10 (or newer) is required now. For older or 32-bit Windows, we can point users to the previous version.

**Reply by jlindgren90** [reply]
Windows build are now linked on the download page as well.

**Comment by carlo-bramini** [comment]
Compiled audacious-4.4-beta1 and all its plugins for CYGWIN.
Configured with  `-D qt5=true` option since only QT5 is available.
Tested both winamp and normal interfaces for both QT and GTK+.
No problem found and everything worked as expected.

**Comment by mschwendt** [comment]
About Meson:

Migrating the Fedora packages to building with Meson has been straight-forward.

**Comment by radioactiveman** [comment]
Tomorrow I will have time to update the translations and to release Audacious 4.4.

@jlindgren90: Are you fine with that? Do you want to do any last checks or tasks before the release?

**Reply by jlindgren90** [reply]
Sounds good, go ahead with the release. I will create a Windows build when I can.

**Reply by radioactiveman** [reply]
Audacious 4.4 is now officially released, see our [announcement](https://audacious-media-player.org/news/59-audacious-4-4-released) or the [GitHub page](https://github.com/audacious-media-player/audacious/releases/tag/audacious-4.4). :partying_face:

**Reply by carlo-bramini** [reply]
Precompiled packages of audacious-4.4 for CYGWIN can found here:
https://github.com/carlo-bramini/packages-cygwin/tree/main/audacious
https://github.com/carlo-bramini/packages-cygwin/tree/main/audacious-plugins
